ui/src/notebooks/pipes/Example/view.tsx

```diff
@@ -0,0 +1,31 @@
+import React, {FC, useContext} from 'react'
+import {NotebookContext} from 'src/notebooks/context/notebook'
+
+import Panel from 'src/notebooks/components/Panel'
+
+interface PipeProps {
+  idx: number
```

Comment:
nit: any opposition to the word `index`? 😆

Reply:
think of my RSI bro